migrations/2020_01_14_130500_change_permission_groups_add_is_hidden.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Please change the filename to have a recent number. (Not 100% sure how well we deal with adding migrations with timestamps older than the currently highest one.) Not super relevant for normal beta upgrades, but possibly for developers / bleeding edge platforms.